lib/worker/metrics-job.js

```diff
@@ -1,15 +1,52 @@
 const statsd = require('../config/statsd')
 const { Subscription } = require('../../lib/models')
 
-module.exports = async (job) => {
-  const syncStatusCounts = await Subscription.syncStatusCounts()
+const ageThreshold = 7
 
-  syncStatusCounts.forEach((row) => {
-    statsd.gauge('syncs', row.count, { status: row.syncStatus })
+function recordBin (bin, label) {
+  bin.forEach((row) => {
+    statsd.gauge(`aged_syncs_bins`, row.count, { status: row.syncStatus, bin: label })
   })
+}
+
+module.exports = async (job) => {
+  Promise.all([
+    Subscription.syncStatusCounts(),
+    Subscription.agedSyncCounts(`${ageThreshold} days`),
+    Subscription.agedSyncCounts('1 second', '2 hours'),
+    Subscription.agedSyncCounts('2 hours', '6 hours'),
+    Subscription.agedSyncCounts('6 hours', '24 hours'),
+    Subscription.agedSyncCounts('1 day', '3 days'),
+    Subscription.agedSyncCounts('3 day', '7 days'),
+    Subscription.agedSyncCounts('7 days', '14 days'),
+    Subscription.agedSyncCounts('14 days', '10 years') // 10 years is a placeholder for "the rest of history"
```

Comment:
These bins may shift (along with the metric name), but I think this is a good start to break down the groups of syncs we're looking for.